MHArmory.Search/OpenCL/search.cl

```diff
@@ -1,236 +1,243 @@
-	#define SKILLS_PER_EQUIPMENT 3
+    #define SKILLS_PER_EQUIPMENT 3
 #endif
 
 #ifndef SET_SKILLS_PER_EQUIPMENT
-	#define SET_SKILLS_PER_EQUIPMENT 3
+    #define SET_SKILLS_PER_EQUIPMENT 3
 #endif
 
 #ifndef EQUIPMENT_TYPES
-	#define EQUIPMENT_TYPES 6
+    #define EQUIPMENT_TYPES 6
 #endif
 
-typedef signed char             int8_t;
-typedef unsigned char 	        uint8_t;
-typedef signed short            int16_t;
-typedef unsigned short 	        uint16_t;
-typedef signed int              int32_t;
-typedef unsigned int     	    uint32_t;
-typedef signed long long int 	int64_t;
-typedef unsigned long long int 	uint64_t;
+typedef signed char int8_t;
+typedef unsigned char uint8_t;
+typedef signed short int16_t;
+typedef unsigned short uint16_t;
+typedef signed int int32_t;
+typedef unsigned int uint32_t;
+typedef signed long long int int64_t;
+typedef unsigned long long int uint64_t;
 
 typedef struct __attribute__((packed)) struct_skill_t
 {
-	int8_t id;
-	int8_t level;
+    int8_t id;
+    int8_t level;
 } skill_t;
 
 typedef struct __attribute__((packed)) struct_set_skill_t
 {
-	uint8_t id;
-	uint8_t requirement;
-	skill_t skill;
+    uint8_t id;
+    uint8_t requirement;
+    skill_t skill;
 } set_skill_t;
 
 typedef struct __attribute__((packed)) struct_set_skill_progress_t
 {
-	uint8_t progress;
-	uint8_t requirement;
-	skill_t skill;
+    uint8_t progress;
+    uint8_t requirement;
+    skill_t skill;
 } set_skill_progress_t;
 
 typedef struct __attribute__((packed)) struct_equipment_t
 {
-	uint16_t id;
-	uint16_t original_id;
-	skill_t skills[SKILLS_PER_EQUIPMENT];
-	set_skill_t set_skills[SET_SKILLS_PER_EQUIPMENT];
-	int8_t slots[SLOT_ARR_LEN];
+    uint16_t id;
+    uint16_t original_id;
+    skill_t skills[SKILLS_PER_EQUIPMENT];
+    set_skill_t set_skills[SET_SKILLS_PER_EQUIPMENT];
+    int8_t slots[SLOT_ARR_LEN];
 } equipment_t;
 
 typedef struct __attribute__((packed)) struct_deco_t
 {
-	uint8_t id;
-	uint16_t original_id;
-	int8_t slots;
-	uint8_t available;
-	skill_t skill;
+    uint8_t id;
+    uint16_t original_id;
+    int8_t slots;
+    uint8_t available;
+    skill_t skill;
 } deco_t;
 
 typedef struct __attribute__((packed)) struct_header_t
 {
-	int8_t weapon_slots[SLOT_ARR_LEN];
-	uint8_t equipment_counts[EQUIPMENT_TYPES];
-	uint8_t deco_count;
-	int8_t desired_skill_count;
-	int8_t set_skill_count;
+    int8_t weapon_slots[SLOT_ARR_LEN];
+    uint8_t equipment_counts[EQUIPMENT_TYPES];
+    uint8_t deco_count;
+    int8_t desired_skill_count;
+    int8_t set_skill_count;
 } header_t;
 
 typedef struct __attribute__((packed)) struct_deco_usage_t
 {
-	uint16_t original_id;
-	int8_t count;
+    uint16_t original_id;
+    int8_t count;
 } deco_usage_t;
 
 typedef struct __attribute__((packed)) struct_result_t
 {
-	uint16_t equipment_ids[6];
-	uint8_t deco_count;
-	deco_usage_t decos[(EQUIPMENT_TYPES + 1) * 3];
+    uint16_t equipment_ids[6];
+    uint8_t deco_count;
+    deco_usage_t decos[(EQUIPMENT_TYPES + 1) * 3];
```

Comment:
Still some hard-coded constants. I'm OK with that not changing them to `const`s but in this case please write a comment before. This will be as good for me, next contributors, or the future you ;)